app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -158,18 +167,9 @@ export function bundlerOptions(args: Arguments) {
             GIT_HASH: JSON.stringify(git('rev-parse HEAD')),
             GIT_STATUS: JSON.stringify(git('status --short --porcelain')),
             BUILD_INFO: JSON.stringify(BUILD_INFO),
-            /** Whether the application is being run locally. This enables a service worker that
-             * properly serves `/index.html` to client-side routes like `/login`. */
```

Comment:
Seems like this comment was lost? Might be useful to keep it in `globals.ts`

Reply:
no longer relevant since we no longer use a service worker anywhere, i think

Reply:
👍